packages/css-library/src/stylesheets/elements.scss

```diff
@@ -5,3 +5,158 @@
 // p {
 //    @include @usa-paragraph
 // }
+
+@use "../../dist/tokens/scss/variables" as *;
+
+$color-link-default-hover: rgba($color-black, 0.05);
```

Comment:
I mentioned this in the [other PR](https://github.com/department-of-veterans-affairs/component-library/pull/981/files#r1423104505) but here it is again. How do add this new color token to the variables build?

Reply:
yeah the opacity thing is tricky. I wonder if there's a `uswds-system-color-grey--` token out there that has a rgba value already converted to hex. Or maybe there's a way to update our build script to support something like that.

Reply:
We might be able to use some sort of style dictionary transform - https://github.com/amzn/style-dictionary/blob/main/examples/advanced/transitive-transforms/tokens/color/overlay.json5

https://amzn.github.io/style-dictionary/#/transforms?id=transitive-transforms

Reply:
@micahchiang I just found that too! 😂 Looks promising!

Reply:
Looking at the transform approach, I think that might complicate things beyond what we need. If we were needing to transform the composed rgba into any other color representation, then that sounds like the right approach.

But just composing it like this seems to be working fine so I think this might be the way to go for now:

https://github.com/department-of-veterans-affairs/component-library/blob/b27e5104f18d7f4eea8aba7d10f495ba0ed2db57/packages/css-library/tokens/color.json#L7-L11

Reply:
Very nice! Definitely the way to go here :+1: out of curiosity, could we also do something like 

```css
"link-default-hover": {
  "alpha": "0.05"
  "value": "rgba({uswds-system-color-gray-100}, {color.link-defaul-hover.alpha})"
}
```
or, could we do something like this where this is the converted hex equivalent for `rgba(0, 0, 0, 0.05)`

```css
"link-default-hover": {
  "value": "{uswds-system-color-gray-100}0d"
}
```

Reply:
>  out of curiosity, could we also do something like
> 
> ```css
> "link-default-hover": {
>   "alpha": "0.05"
>   "value": "rgba({uswds-system-color-gray-100}, {color.link-defaul-hover.alpha})"
> }
> ```

Yes! Good call. I like using the USWDS black instead. 


> or, could we do something like this where this is the converted hex equivalent for `rgba(0, 0, 0, 0.05)`
> 
> ```css
> "link-default-hover": {
>   "value": "{uswds-system-color-gray-100}0d"
> }
> ```

No, I don't believe this is the same thing. Assigning this as just a hex value means that a `transition-property` will not work. It needs the `alpha` value to display the fade effect going from black to 0.05 of black.

Reply:
As a sanity check, I tested using a hex value instead of `0,0,0` for rgba in vets-website because the[ rgba function specs don't show hex as an example](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/rgb):

![Screenshot 2023-12-14 at 8 54 20 AM](https://github.com/department-of-veterans-affairs/component-library/assets/872479/f35818c4-b387-4093-8c1f-84b492f4d51b)

![Screenshot 2023-12-14 at 9 19 33 AM](https://github.com/department-of-veterans-affairs/component-library/assets/872479/2da0aee5-9969-4f58-820e-1b48622ac3dc)

It looks like `rgba` automatically converts the hex to a red/green/blue value so I think using `uswds-system-color-gray-100` instead of 0,0,0 will be fine. 

And the support for hex color notation looks good (except for IE which we no longer support anyway): https://caniuse.com/css-rrggbbaa

**Chrome**
![Screenshot 2023-12-14 at 9 18 18 AM](https://github.com/department-of-veterans-affairs/component-library/assets/872479/76eb67f1-4898-4cda-bf5e-181c7db51acb)